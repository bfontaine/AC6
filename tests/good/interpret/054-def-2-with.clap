def  f (x) = if x < 0 then 42 else g (x)
with g (x) = f(x-1)

val q = f(42)

