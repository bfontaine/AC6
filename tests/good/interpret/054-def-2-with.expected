:- f = <code>
:- g = <code>
:- q = 42
